include/ode_solvers/leapfrog.hpp

```diff
@@ -10,6 +10,7 @@
 
 #ifndef ODE_SOLVERS_LEAPFROG_HPP
 #define ODE_SOLVERS_LEAPFROG_HPP
+#include "Eigen/Eigen"
```

Comment:
everywhere in the code, we have used `<>` for eigen and boost, so I think we should keep it consistent.